awx/main/models/mixins.py

```diff
@@ -247,15 +247,15 @@ def _survey_element_validation(self, survey_element, data, validate_required=Tru
                 else:
                     choice_list = copy(survey_element['choices'])
                     if isinstance(choice_list, str):
-                        choice_list = choice_list.split('\n')
+                        choice_list = [choice for choice in choice_list.split('\n') if choice.strip() != '']
```

Comment:
I'd say that this line and the one below should get the same treatment (i.e. `.splitlines()`) as used in the other file.

Reply:
:woman_facepalming: on it :)